common/OpTestUtil.py

```diff
@@ -181,3 +181,102 @@ def copyFilesToDest(
 
         print(list)
         return list
+
+    ##
+    #   @brief    This method does a scp from remote system (where files are found)
+    #             to destination(Path where files will be stored) or local system.
+    #   @param    destid    userid of remote system
+    #   @param    destName  ip of remote system
+    #   @param    destPath  file path in remote system
+    #   @param    hostPath  exact file path to store into local system
+    #   @param    passwd    password of remote system
+    #   @param    ssh_ver
+    #   @return   output from terminal
+    #   @throw    subprocess or OpTestError
+    #
+    def copyFilesFromDest(
+            self,
+            destid,
+            destName,
+            destPath,
+            hostPath,
+            passwd,
+            ssh_ver="2"):
+        pid, fd = pty.fork()
+        Password = passwd + "\r\n"
+        list = ''
+        destinationPath = destid.strip() + "@" + destName.strip() + \
+            ":" + destPath
+
+        # We've spawned a separate process with the .fork above so one process
+        # will execute the scp command, and the other process will handle the
+        # back and forth of the password and the error paths
+        if pid == 0:
+            arglist = (
+                "/usr/bin/scp",
+                "-" +
+                ssh_ver.strip(),
+                destinationPath,
+                hostPath)
+            print(arglist)
+            os.execv("/usr/bin/scp", arglist)
+        else:
+            while True:
```

Comment:
This is a copy/paste of the above code?  Should be a common function they both call.